tasks/main.yml

```diff
@@ -1,6 +1,10 @@
 ---
-- name: Include OS-specific variables.
-  include_vars: "{{ ansible_os_family }}.yml"
+- name: set ntp_daemon to 'ntp' as default
```

Comment:
I'd rather keep this as a distro-specific included variable instead—it's a pattern I'm using across all my roles, and it keeps it a little more simple.

If the variable needs to be able to be overridden, we can make it `__ntp_daemon` and set it at runtime like I do in roles like `apache` and `php`.

If it's just a matter of adding more OSes, I'd rather be explicit and add the `vars` files per-OS.